### PR TITLE
made the navbar sticky while scrolling

### PR DESCRIPTION
**Pull request by anubhav1206**

### CODE REVIEW

**Comment by nitin-pandita** [comment]
@anubhav1206  please attach a screenshot

File: style.css [hunk 1]
```diff
@@ -7,11 +7,30 @@
 body {
     font-family: 'Poppins', sans-serif;
 }
+
+.logo{
```
Comment (nitin-pandita) [inline style.css:11]: Please add a background when scrolling to make it look good

File: style.css [hunk 2]
```diff
@@ -7,11 +7,30 @@
 body {
     font-family: 'Poppins', sans-serif;
 }
+
+.logo{
+    position: fixed;
+    top: 0;
+    left: 0;
+}
 .navbar {
```
Comment (nitin-pandita) [inline style.css:16]: The Cart icon is not showing please fix that too

**Review by nitin-pandita — CHANGES_REQUESTED** [review]
Add a background behind the navbar and fix the cart icon

File: style.css (hunk 2 above)
Comment (anubhav1206) [reply style.css:16]: will make the necessaryy fixes

**Comment by anubhav1206** [comment]
Here is the screenshot for the same
image

**Comment by nitin-pandita** [comment]
> Here is the screenshot for the same image

website

It is overlapping the elements please fix that

**Comment by anubhav1206** [comment]
I tried but I am unable to come up with a fix for the overlapping logo.

**Comment by nitin-pandita** [comment]
> I tried but I am unable to come up with a fix for the overlapping logo.

you can take an example from this code
https://github.com/nitin-pandita/Nitin-Fashionwear/pull/15
He has done a that

**Pull request closed without merging**